src/server/game/Anticheat/AnticheatMgr.cpp

```diff
@@ -230,6 +230,16 @@ void AnticheatMgr::ZAxisHackDetection(Player* player, MovementInfo movementInfo)
    if (m_Players[key].GetLastMovementInfo().HasMovementFlag(MOVEMENTFLAG_WATERWALKING) && movementInfo.HasMovementFlag(MOVEMENTFLAG_WATERWALKING))
        return;
 
+   switch (player->GetAreaId())
+   {
+   case 4281: //Acherus: The Ebon Hold
+   case 4342: //Acherus: The Ebon Hold
+       return;
+       break;
+   default:
+       break;// Should never happen
```

Comment:
What's the comment "should never happen" about ?

Reply:
these were notes left on the counter core that i am maintaining now. So i pretty much been doing copy and pasting when select cases are needed\required atleast at the time.